docker-compose.prod.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.7'
+services:
+  bot:
+    image: godfather:alpha
+    restart: "always"
+    command: "start"
+    env_file: .env
+    ports:
+        - "4000:8080"
+    depends_on:
+        - postgres
+  postgres:
+    image: postgres
+    restart: "always"
+    env_file: .env
+    ports:
+        - "$PGSQL_DATABASE_EXPOSED_PORT:5432"
+    environment:
+        - POSTGRES_DB=godfather
+        - "POSTGRES_USER=$PGSQL_DATABASE_USER"
```

Comment:
the `$` prefixed are env variables defined in your `.env` file. It will the use the value from there